### PR TITLE
[MM-49344] - Improve error message when trying to upload another trial license

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR adds a clearer message when a user tries to upload a second trial license. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-49344

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by neallred — APPROVED** [review]

File: i18n/en.json [hunk 1]
```diff
@@ -2055,7 +2055,7 @@
   },
   {
     "id": "api.license.request-trial.can-start-trial.not-allowed",
-    "translation": "This trial license key for Mattermost Enterprise Edition has expired and is no longer valid. If you would like to extend your trial period please [contact our sales team](https://mattermost.com/contact-us/)."
+    "translation": "Failed to apply new trial license. You have previously applied a trial license to this Mattermost instance.. If you would like to extend your trial period please [contact our sales team](https://mattermost.com/contact-us/)."
```
Comment (marianunez) [inline i18n/en.json:2058]: Extra point typo
```suggestion
    "translation": "Failed to apply new trial license. You have previously applied a trial license to this Mattermost instance. If you would like to extend your trial period please [contact our sales team](https://mattermost.com/contact-us/)."
```

**Review by marianunez — APPROVED** [review]
Thanks @AGMETEOR

**Comment by marianunez** [comment]
@amyblais small change but nice to have for v7.7 if possible to reduce confusion with why trial licenses are not uploaded

**Pull request merged**